test/CDFDocumentTest.php

```diff
@@ -0,0 +1,428 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: eugene
+ * Date: 2/18/19
+ * Time: 3:04 PM
+ */
```

Comment:
We don't have a standard to add the line 2 - 7. Before we start this new tradition, we have to get team consensus plus everyone to do the same. For now, please remove line 2 to 7 and stop your IDE from generating this automatically for Content Hub files. ^^

Reply:
Still pending.